build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.15 AS builder
+FROM quay.io/app-sre/boilerplate:image-v0.2.0 AS builder
```

Comment:
Please do the same thing with `Dockerfile.test`

Reply:
Okay. For now this will be "manual", not managed by boilerplate (which means it'll have to be manually kept in sync in future bp updates). I'm pretty sure we don't want to go around indiscriminately changing all Dockerfiles, because they don't necessarily want to start with our image. Perhaps we can come up with some kind of rule, like `build/Dockerfile*` gets changed but anything else (other dirs, or other name patterns) doesn't? Thoughts?

Reply:
Done via c1219ce.

Reply:
`deploy/bundle/Dockerfile` will soon disappear as I intend to use `opm` to create the bundle directly so we don't have to keep that Dockerfile so I guess the rule to just change `build/Dockerfile*` makes sense